lib/msal-common/src/server/ServerCodeRequestParameters.ts

```diff
@@ -29,12 +29,15 @@ export class ServerCodeRequestParameters extends ServerRequestParameters {
     // Validity checks
     nonce: string;
 
-    constructor(authority: Authority, clientId: string, userRequest: AuthenticationParameters, redirectUri: string, cryptoImpl: ICrypto, isLoginCall: boolean) {
+    // Account
+    account: Account;
+
+    constructor(authority: Authority, clientId: string, userRequest: AuthenticationParameters, cachedAccount: Account, redirectUri: string, cryptoImpl: ICrypto, isLoginCall: boolean) {
         super(clientId, redirectUri, cryptoImpl);
         this.authorityInstance = authority;
         this.userRequest = userRequest;
-
         this.responseType = Constants.CODE_RESPONSE_TYPE;
+        this.account = (userRequest && userRequest.account) || cachedAccount;
```

Comment:
When would `userRequest` be falsey?

Reply:
In a login request, you are allowed to send a null or empty request object.